Sources/App/Controllers/LoginController.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  LoginController.swift
+//  App
+//
+//  Created by Tiewhan Smith on 2020/04/07.
+//
+
+import Foundation
+import Vapor
+
+struct LoginCredentials: Content {
+  let username: String
+  let password: String
+}
+
+class LoginController {
+  
+  func verifyLoginCredentials(_ loginCredentials: LoginCredentials) -> String {
+    
+    if loginCredentials.username == "tiewhan" && loginCredentials.password == "cat3" {
```

Comment:
This is fine really for server-side since it's not too major but I'm curious whether you plan to support multiple users in the future ? Or just the one ?

Reply:
For now, it's one. Until I get other things sorted it will stay like this. Get the meat of the application ready